assets/deploy-nfs.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Clone external-storage repo for NFS provisioner templates
+git clone https://github.com/kubernetes-incubator/external-storage
+
+# Modify StorageClass to be our default (add annotation)
+echo '---
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: example-nfs
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+provisioner: example.com/nfs
```

Comment:
Can we make this something other than "example"?

Reply:
This will require further testing... anytime I try to change a provisioner name (on any of the provisioners), it usually results in problems. I do agree that `example.com` is less than desirable.